plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -3193,6 +3355,27 @@ def main():
     else:
         ccc_network.log("No reboot list found in validated configuration.", "WARNING")
 
+    # Below set of line related to the Access Point factroy reset function.
```

Comment:
Do we need to include all of these statements within the main function? We intended to maintain the module structure throughout our workflow modules. However, this AP workflow manager module appears to be distinct from the others.

Reply:
created new function and moved.